### ISSUE TITLE
Support: VoiceOver: alerts are not accessible and Contact Us screen should have instructions

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12177]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. Go to Profile (Me) > Help & Support for a new account.
1. You will see an alert either asking to update email & username or an a8c-only warning.
1. ⚠️ The contents of the alert is not read out by VoiceOver. 
1. Go to Contact Us.
1. ⚠️ No instructions are provided for the Contact Us screen.
1. Fill out the "Write a message…" field and double tap the Send button.
1. ⚠️ After sending, a success message appears but is not read out by VoiceOver. The message says, "Your message has been sent. If a reply is needed, we will contact you shortly."

**Result:** alerts in the Profile (Me) section are not accessible and, the Contact Us screen is missing instructions, and the success notice after sending a message is not read out by VoiceOver. ([4min](https://cloudup.com/cBj3GDv4byp))

<img src="https://user-images.githubusercontent.com/1119271/61649886-97d24d00-ac6f-11e9-9700-65e5a8d1d9bb.png" width="200" alt="please-enter-your-email-address"> <img src="https://user-images.githubusercontent.com/1119271/61649885-97d24d00-ac6f-11e9-86bc-6d6ec5f65651.png" width="200" alt="contact-us"> <img src="https://user-images.githubusercontent.com/1119271/61649883-97d24d00-ac6f-11e9-97da-62462f6fd17b.png" width="200" alt="contact-us-success-message">
<sup>Tested with WPiOS 12.8.1 on iPhone 6S iOS 12.3.1.</sup>

<p style="clear:both;"><sub>(internal reference: p1563664437001300-slack-cellular-happiness, p77Llu-bQ6-p2#comment-1307) cc @annacmc</sub></p>
